src/videotile/DefaultVideoElementResolutionMonitor.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import VideoElementResolutionMonitor, {
+  VideoElementResolutionObserver,
+} from './VideoElementResolutionMonitor';
+export default class DefaultVideoElementResolutionMonitor implements VideoElementResolutionMonitor {
+  private observerQueue = new Set<VideoElementResolutionObserver>();
+  private resizeObserver: ResizeObserver;
+  private element?: HTMLVideoElement;
+
+  constructor(element?: HTMLVideoElement) {
```

Comment:
technically don't need this constructor arg.

Reply:
Removed